src/connectors/toggle-refinement/connectToggleRefinement.js

```diff
@@ -185,7 +206,9 @@ export default function connectToggleRefinement(renderFn, unmountFn = noop) {
       },
 
       render({ helper, results, state, instantSearchInstance }) {
-        const isRefined = helper.state.isDisjunctiveFacetRefined(attribute, on);
+        const isRefined =
```

Comment:
similar to this, we also need to edit it in `getWidgetState`: ?instant_search%5BrefinementList%5D%5Bbrand%5D%5B0%5D=Sony&instant_search%5BrefinementList%5D%5Bbrand%5D%5B1%5D=Incipio

Reply:
Oh, thank you!